config/routes.rb

```diff
@@ -25,5 +26,4 @@
   end
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
If you are deleting line 28 you could remove lines 26 and 27 as well since they are unnecessary.

Reply:
Just remove the lines based on your recommendation and updated.